Sources/SwiftFormatConfiguration/Configuration.swift

```diff
@@ -88,6 +88,15 @@ public class Configuration: Codable {
   /// each argument, forcing the entire argument list to be laid out vertically.
   public var lineBreakBeforeEachArgument = false
 
+  /// Determines if function-like declaration outputs should be prioritized to be together with the
+  /// function signature right (closing) parenthesis.
+  ///
+  /// If false (the default), function output (i.e. throws, return type) is not prioritized to be
+  /// together with the signature's right parenthesis, and when the line length would be exceeded,
+  /// a line break will be fired after the function signature first, indenting the declaration output
+  /// one additional level. If true, A line break will be fired further up in the function's
+  /// declaration (e.g. generic parameters, parameters) before breaking on the function's output.
+  public var prioritizeKeepingFunctionOutputTogether = false
```

Comment:
We can't use synthesized `Codable` because we have some custom logic in our `Configuration` type, so you'll also need to add this property manually to the `CodingKeys` enum and to the `init(from: Decoder)` and `encode(to: Encoder)` methods.

Reply:
Completely missed that, thanks!